pagopa/io-app
# fix: [IOPAE-1164] Fix infinite scroll in `InstitutionServicesScreen`

## Short description
This PR fixes the infinite scroll in `InstitutionServicesScreen` by removing external `ScrollView` because the `FlatList` inside animated `ScrollView` causes infinite triggering of the `onEndReached` method without even scrolling.

## How to test
Using `io-dev-api-server`, navigate to the services tab and tap an institution. Check that infinite scroll works correctly and that `onEndReached` is triggered only when scrolling to the bottom of the list.
